### DISCUSSION TITLE
backend setup issues ..!!

### DISCUSSION THREAD

**Discussion by vivekpanchal** [PatilShreyas/NotyKT#686] [Discuss noty-api (Backend development)]
Hey i have cloned the project and followed the steps that are mentioned in pages 
1 .cloned the repo
2. installed postgres and created the database **notykt_dev_db**
3. in .env  i have replaced with my properties 

```
SECRET_KEY=ANY_RANDOM_SECRET_VALUE

PGPORT=5432
PGHOST=localhost
PGDATABASE=notykt_dev_db
PGUSER=postgres
PGPASSWORD=1234

DATABASE_DRIVER=org.postgresql.ds.PGSimpleDataSource
DATABASE_MAX_POOL_SIZE=10
```

still i am getting error for PGPASSWORD

` Could not resolve substitution to a value: ${PGPASSWORD}
`

am I missing something? @Spikeysanju  @PatilShreyas  ?

**Comment by PatilShreyas** [comment]
@vivekpanchal you don't have to set these values in .env file. Instead, these values should be available as environment variables.

**Reply by vivekpanchal** [reply]
Thanks i got it
